tick/base/simulation/simu_with_features.py

```diff
@@ -144,22 +148,24 @@ def features_scaling(self, val):
             val = "none"
         self._set("_features_scaling", val)
 
-    def simulate(self):
+    def simulate(self, dtype="float64"):
```

Comment:
I would give `dtype` to the constructor and document it by telling that it would be overriden by `features.dtype` if `features` is given